Podcast/OnboardingButton.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  OnboardingButton.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/2/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+/// semi-transparent buttons used in onboarding view
+
+class OnboardingButton: UIButton {
+
+    convenience init(title: String) {
+        self.init(frame: .zero, title: title)
+    }
+
+    init(frame: CGRect, title: String) {
+        super.init(frame: frame)
+        backgroundColor = UIColor.offWhite.withAlphaComponent(0.15)
```

Comment:
.offWhite? Or does this not work due to the method call

Reply:
Have a feeling because you mixed .offWhite and UIColor.offWhite that there's some usage of properties attached that makes removing the UIColor not possible

Reply:
expression is ambiguous